avaframe/ana3AIMEC/ana3AIMEC.py

```diff
@@ -221,66 +208,89 @@ def postProcessAIMEC(cfg, rasterTransfo, pathDict, inputsDFrow, newRasters, time
     cfgFlags = cfg['FLAGS']
     interpMethod = cfgSetup['interpMethod']
     flagMass = cfgFlags.getboolean('flagMass')
-    refSimulationName = pathDict['refSimulation']
+    refSimRowHash = pathDict['refSimRowHash']
     resTypeList = pathDict['resTypeList']
-    # apply domain transformation
 
+    # apply domain transformation
     log.info('Analyzing data in path coordinate system')
 
     for resType in resTypeList:
         log.debug("Assigning %s data to deskewed raster" % resType)
-        inputFiles = inputsDFrow[resType]
-        rasterData = IOf.readRaster(inputFiles)
-        newRaster = aT.transform(rasterData, inputFiles, rasterTransfo, interpMethod)
-        newRasters['newRaster' + resType.upper()] = newRaster
-        if simName == refSimulationName:
-            newRasters['newRefRaster' + resType.upper()] = newRaster
-            resAnalysisDF[resType + 'CrossMax'] = np.nan
-            resAnalysisDF[resType + 'CrossMax'] = resAnalysisDF[resType + 'CrossMax'].astype(object)
-            resAnalysisDF[resType + 'CrossMean'] = np.nan
-            resAnalysisDF[resType + 'CrossMean'] = resAnalysisDF[resType + 'CrossMax'].astype(object)
-
-        # analyze all fields
-        resAnalysisDF = aT.analyzeField(simName, rasterTransfo, newRaster, resType, resAnalysisDF)
-
-    # compute runout based on resType
-    resAnalysisDF = aT.computeRunOut(cfgSetup, rasterTransfo, resAnalysisDF, newRasters, simName)
+        inputFiles = resAnalysisDF.loc[simRowHash, resType]
+        if isinstance(inputFiles, pathlib.PurePath):
+            rasterData = IOf.readRaster(inputFiles)
+            newRaster = aT.transform(rasterData, inputFiles, rasterTransfo, interpMethod)
+            newRasters['newRaster' + resType.upper()] = newRaster
+            if simRowHash == refSimRowHash:
+                newRasters['newRefRaster' + resType.upper()] = newRaster
+                resAnalysisDF[resType + 'CrossMax'] = np.nan
```

Comment:
I first need to create the column, this could be done using and empty string "" too or using the pandas insert function. I do not know what is best...